aten/src/ATen/ConjugateFallback.cpp

```diff
@@ -18,6 +18,10 @@ TORCH_LIBRARY_IMPL(_, Conjugate, m) {
   m.fallback(torch::CppFunction::makeFromBoxedFunction<&conjugateFallback>());
 }
 
+TORCH_LIBRARY_IMPL(prims, Conjugate, m) {
+  m.impl("clone", torch::CppFunction::makeFallthrough());
+}
```

Comment:
Add test please?

Reply:
will do!

Reply:
turns out that this wasn't even needed (I had to fix https://github.com/pytorch/pytorch/issues/103532 instead)